src/status_im/contexts/wallet/add_account/create_account/new_keypair/confirm_backup/view.cljs

```diff
@@ -32,52 +33,72 @@
     [(subvec result 0 2) (subvec result 2 4)]))
 
 (defn- cheat-warning
-  [on-try-again]
+  [on-try-again shell?]
   (let [customization-color (rf/sub [:profile/customization-color])]
     [:<>
      [quo/drawer-top {:title (i18n/label :t/do-not-cheat)}]
-     [quo/text
-      {:style style/cheat-description}
+     [quo/text {:style style/cheat-description}
       (i18n/label :t/do-not-cheat-description)]
      [quo/bottom-actions
-      {:button-one-label (i18n/label :t/see-recovery-phrase-again)
+      {:actions          :one-action
+       :button-one-label (i18n/label :t/see-recovery-phrase-again)
        :button-one-props {:customization-color customization-color
                           :on-press            (fn []
                                                  (rf/dispatch [:hide-bottom-sheet])
                                                  (rf/dispatch [:navigate-back])
                                                  (when on-try-again
-                                                   (on-try-again)))}}]]))
+                                                   (on-try-again)))}
+       :blur?            shell?}]]))
 
 (defn- button
-  [{:keys [word margin-right on-press]}]
+  [{:keys [word margin-right on-press shell?]}]
   [quo/button
    {:type            :grey
     :on-press        #(on-press word)
-    :container-style (style/button margin-right)} word])
+    :container-style (style/button margin-right)
+    :background      (when shell? :blur)}
+   word])
 
 (defn- buttons-row
-  [{:keys [margin-bottom options] :as params}]
+  [{:keys [margin-bottom options shell?] :as params}]
   [rn/view {:style (style/buttons-inner-container margin-bottom)}
    [button
     (assoc params
            :word         (first options)
-           :margin-right 12)]
-   [button (assoc params :word (second options))]])
+           :margin-right 12
+           :shell?       shell?)]
+   [button
+    (assoc params
+           :word   (second options)
+           :shell? shell?)]])
 
 (defn- complete-backup-sheet
-  [on-success on-try-again]
+  [on-success on-try-again shell?]
   (let [customization-color    (rf/sub [:profile/customization-color])
-        [checked? set-checked] (rn/use-state false)]
+        [checked? set-checked] (rn/use-state false)
+        change-checked         (rn/use-callback #(set-checked not))]
     [:<>
      [quo/drawer-top {:title (i18n/label :t/complete-backup)}]
      [quo/text
       {:style style/cheat-description}
       (i18n/label :t/ensure-written-recovery)]
-     [quo/disclaimer
-      {:checked?        checked?
-       :container-style {:margin-horizontal 20}
-       :on-change       #(set-checked (not checked?))}
-      (i18n/label :t/written-seed-ready)]
```

Comment:
We were using a disclaimer but the designs show something different :eyes: it's been changed